cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -68,6 +68,23 @@
     }
 }
 
+.o-video-player_duration {
+    color: rgb( 255, 255, 255 );
```

Comment:
`@white`?

Reply:
Updated in https://github.com/cfpb/consumerfinance.gov/pull/6267/commits/8fa1b3dd4e47337e2002c1fec103197efbbb9f10